aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -97,17 +95,17 @@ Tensor nested_from_padded_cuda(
             offsets_ptr,
             input_size_ptr,
             output_size_ptr,
-            padded_contiguous.dim() - 2,
-            padded_contiguous.sizes()[0]);
+            static_cast<int>(padded_contiguous.dim() - 2),
+            static_cast<int>(padded_contiguous.sizes()[0]));
       } else {
         remove_padding_kernelLauncher(
             padded_contiguous.data_ptr<c10::Half>(),
             output.data_ptr<c10::Half>(),
             offsets_ptr,
             input_size_ptr,
             output_size_ptr,
-            padded_contiguous.dim() - 1,
-            padded_contiguous.sizes()[0]);
+            static_cast<int>(padded_contiguous.dim() - 1),
+            static_cast<int>(padded_contiguous.sizes()[0]));
```

Comment:
Can't this causs isues without large nested tensors elements (with >int32 elements)?

Reply:
Need to change API

Reply:
It's probably masking a bug though, is the API actually public and stable? Otherwise I'd advocate to change it for prevent silent bugs.

Reply:
@cyyever Yeah, doesn't seem to be marked with `TORCH_API` so we can change it.

Reply:
Fixed